threedi_schema/beta_features.py

```diff
@@ -0,0 +1,48 @@
+from .domain import constants, models
+
+"""
+Put features in beta development in these lists to prevent users from using them, in the SQLAlchemy format used in the schema.
+"""
+"""This list contains beta tables, e.g. models.SomeTable"""
+BETA_TABLES = [
```

Comment:
The table itself is indeed beta, but I think it is easier to put the "beta" flag on the `models.GlobalSetting.vegetation_id`. Then it falls in the `BETA_COLUMNS` category; so if someone has a record in the vegetation table, that's ok, as long as it is not used.